pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -296,11 +298,11 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 			}
 			return TokenKind(ch), tkn.bytes()
 		case '!':
-			switch tkn.lastChar {
-			case '=':
+			switch c := tkn.lastChar; {
+			case c == '=':
```

Comment:
I assume this change and the one below was unintentionally committed.
```suggestion
			switch tkn.lastChar; {
			case '=':
```

Reply:
It wasn't really unintentional. I needed to change the switch style to call `isValidCharAfterOperator(c)` but I revisited that implementation style after seeing how we just put those in the `default` case in other places in the tokenizer. I updated the PR with that version.

Reply:
Of course 🤦🏻 Thanks for explaining!